src/components/LoginForm.vue

```diff
@@ -44,6 +44,10 @@
 </template>
 
 <script>
+function validatePassphrase (passphrase) {
+  return passphrase.toLowerCase().trim().split(' ').length === 12
```

Comment:
Why do we need toLowerCase here?

Reply:
Also maybe instead of counting spaces we should use bip39.validateMnemonic(mnemonic)?